sdk/template/azure-template-second/vcpkg/CONTROL

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+#
+Source: azure-template-second-cpp
+Version: @AZ_LIBRARY_VERSION@
+Build-Depends: azure-core-cpp
```

Comment:
Is this meant to only depend on core?  I thought template second was a dependency on template.